GVFS/GVFS.Installer.Windows/Setup.iss

```diff
@@ -27,6 +27,7 @@
 #define EnvironmentKey "SYSTEM\CurrentControlSet\Control\Session Manager\Environment"
 #define FileSystemKey "SYSTEM\CurrentControlSet\Control\FileSystem"
 #define GvFltAutologgerKey "SYSTEM\CurrentControlSet\Control\WMI\Autologger\Microsoft-Windows-Git-Filter-Log"
+#define ServiceUIName "GVFS.Service.UI"

```

Comment:
Do we want to give the UI a nicer name (assuming the notification sender is based on the shortcut name), like "VFS for Git"?

![image](https://user-images.githubusercontent.com/5658207/64708160-10a19880-d4ac-11e9-8a40-6428b12c9de8.png)


```suggestion
#define ServiceUIName "VFS for Git"
```

Reply:
Updated it to `VFS For Git`